modules/determining-upgrade-viability-cv-conditiontype.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * updating/index.adoc
+
+:_content-type: CONCEPT
+[id="Understanding_clusterversion_conditiontypes_{context}"]
+= Understanding cluster Version condition types
+ 
+The status of the cluster Version includes the condition type, informing you of the current state of your cluster version Operator's health. 
+
+The Cluster Version Operator (CVO) is responsible for collecting the status conditions from cluster version Operators so that cluster administrators can better understand the state of the {product-title} cluster. 
+
+* Available: 
+An Operator with the condition type `Available` is functional and available in the cluster. If the status is `False`, at least one part of the operand is non-functional and the condition requires an administrator to intervene.
+
+* Progressing:
+An Operator with the condition type `Progressing` is actively rolling out new code, propagating configuration changes, or otherwise moving from one steady state to another. 
++
+Operators do not report the condition type `Progressing` as `True` when they are reconciling a previous known state. If the observed cluster state has changed and the Operator is reacting to it, then the status will report back as `True`, since it is moving from one steady state to another.
+
+* Upgradeable:
+An Operator with the condition type `Upgradeable` indicates if the Operator is safe for you to perform a minor or patch update based on the current cluster state. 
++
+The message field contains a human-readable description of what the administrator needs to do for the cluster to successfully update. The CVO allows updates when this condition is `True`, `Unknown` or missing. 
++
+When the `Upgradeable` status is `False`, minor and patch updates are impacted, and the CVO prevents the cluster from performing impacted updates unless forced.
+
+* Failing: 
+The CVO monitors cluster operators and other components. `ClusterStatusFailing` is set on the cluster Version status when a cluster cannot reach the desired state. It is considered more severe than `Degraded`, indicates that the cluster is not healthy, and requires an administrator to intervene.
```

Comment:
Do we want to distinguish the `ClusterVersion`-only conditions somehow, from the ones that could be also on `ClusterOperator`?

Reply:
@wking and I discussed this a bit.  I added the ones that each one can have in their respective sections, knowing there is a bit of repetition. My reasoning is that a user wouldn't need to read both sections in case they only need conditions for one or the other. @LalatenduMohanty what do you think?

Reply:
The condtion types have similar/same definition when they are present in both sections. So IMO we should remove the repetition and only put condition types under cluster version which do not exist under cluster operator section.